<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>加噪&amp;去噪</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>481</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>高斯噪声</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>算术均值滤波</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>椒盐噪声</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>百分比</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="jyzs_value">
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>高斯滤波</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>均值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="gszs_value1"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>标准差</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="gszs_value2"/>
       </item>
      </layout>
     </item>
     <item row="4" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>m</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="value_jzlb1"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>n(=m)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="value_jzlb2"/>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>模板(y*y)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="value_gslb_ab"/>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>标准差</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="value_gslb_c"/>
       </item>
      </layout>
     </item>
     <item row="2" column="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>中值滤波</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,5">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>孔径的尺寸</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="value_zzlb"/>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Line" name="line_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="Line" name="line_10">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="Line" name="line_11">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="Line" name="line_12">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="Line" name="line_13">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="submit_zzlb">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="submit_jyzs">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="submit_gszs">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="submit_gslb">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="submit_jzlb">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
